src/deployCommand.ts

```diff
@@ -121,7 +121,9 @@ export abstract class DeployCommand extends SourceCommand {
       return false;
     }
 
-    const restDeployConfig = this.configAggregator.getInfo(SfdxPropertyKeys.REST_DEPLOY).value;
+    const restDeployConfig =
```

Comment:
if you run `sf force:source:deploy` the `configAggregator` will be a `ConfigAggregator` not a `SfdxConfigAggregator` which won't know about the `sf` config values